machado/management/commands/load_rnaseq_data_lstrap.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2018 by Embrapa.  All rights reserved.
```

Comment:
what about renaming it to load_rnaseq.py?
Right now it loads lstrap files, but we can generalize this software in the future.